server/src/main/java/edu/mit/csail/db/ml/conf/ModelDbConfig.java

```diff
@@ -156,6 +171,23 @@ private static String getProp(Config config, String key) {
     return config.getString(String.format("modeldb.%s", key));
   }
 
+
+  /**
+   * Read the key "modeldb.[keyname]" from the given configuration object, if it exists.
+   * @param config - The configuration object.
+   * @param key - Name of the key to lookup.
+   * @return The value of the given key, if it exists.
+   */
+  private static Optional<String> getOptionalProp(Config config, String key) {
+    String path = String.format("modeldb.%s", key);
+    if (config.hasPath(path)) {
+      return Optional.of(config.getString(path));
+    } else {
+      return Optional.empty();
+    }
+  }
+
```

Comment:
Remove extra blank line here (and before method).

(There should be one blank line between methods).

Reply:
done